resources/css/main.scss

```diff
@@ -775,15 +775,15 @@ body.home {
           left: 15px;
           color: #fff;
           width: 70%;
-          padding: 15px 0;
+          padding: 8px 0;
```

Comment:
This change and the one below is so that it's not completely ugly to stack up two 10% projects.